server/src/logs.py

```diff
@@ -1,27 +1,63 @@
+
 import logging
 import sys
+
+from typing import cast
+from pathlib import Path
 from logging.handlers import RotatingFileHandler
 
+
 from .config import cfg
-from .utils import FILE_DIR
+from .utils import DATA_DIR
+from .config.types import StandardOutLoggingConfig, FileLoggingConfig
 
 # SETUP LOGGING
 
+config = cfg()
+
+level_mapping = logging.getLevelNamesMapping()
 logger = logging.getLogger("PlanarAllyServer")
-logger.setLevel(logging.INFO)
-file_handler = RotatingFileHandler(
-    str(FILE_DIR / "planarallyserver.log"),
-    maxBytes=cfg().general.max_log_size_in_bytes,
-    backupCount=cfg().general.max_log_backups,
-)
-file_handler.setLevel(logging.INFO)
+
+set_stdout = False
+
 formatter = logging.Formatter("%(asctime)s - %(levelname)s - %(message)s (%(filename)s:%(lineno)d)")
-file_handler.setFormatter(formatter)
-stream_handler = logging.StreamHandler(sys.stdout)
-stream_handler.setFormatter(formatter)
-logger.addHandler(file_handler)
-logger.addHandler(stream_handler)
 
+if not config.logging:
+    # Write to standard out there is no logger configured 
+    sys.stdout.write("No logging configuration found. There will be no logging output.\n")
```

Comment:
You can just use `print()` :D no need for the `\n` either.

Unless there is a specific reason you went for the more low level API, but I assume this is the C#/Java coming through :)